api/filters/imagetag/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package imagetag contains two kio.Filter implementations to cover the
```

Comment:
should this just be `image` since it also updates the name?

Reply:
You thinking that the package should be name `image` rather than `imageTag`? I was thinking that the `Tag` in this context meant a field in a yaml document rather than the `tag` part of the value of the `image` field. I'm open to changing the package name if we think it is better. So far I've just tried to stay consistent with the terms used in the transformers.

Reply:
sgtm